scripts/prep-for-deploy-from-github-actions.sh

```diff
@@ -0,0 +1,20 @@
+export BRANCH=$(
+  if [ -n "$GITHUB_HEAD_REF" ]; then
```

Comment:
This is breaking the continuous deployment script. The branch name seems to come out in the form `refs_heads_<branch-name>` (e.g. `refs_heads_github-actions`). The continuous deployment script is probing the folder with just the branch name instead.

Reply:
Thanks for catching that, fixed in f148f6d.